proxy/src/test/java/com/wavefront/agent/listeners/otlp/OtlpTraceUtilsTest.java

```diff
@@ -1021,4 +1021,19 @@ public void shouldReportSpanLogsTrueIfNonZeroLogsAndFeatureEnabled() {
     Supplier<Boolean> spanLogsFeatureDisabled = () -> false;
     assertTrue(OtlpTraceUtils.shouldReportSpanLogs(1, Pair.of(spanLogsFeatureDisabled, null)));
   }
+
+  @Test
+  public void testGetAttrKeyValue() {
+    KeyValue attrKeyValue = OtlpTraceUtils.getAttrKeyValue("test_key", "test_value");
+    List<KeyValue> attributes = new ArrayList<>();
+    attributes.add(attrKeyValue);
+    assertEquals("test_value", OtlpTraceUtils.getAttrValByKey(attributes, "test_key"));
+  }
+
+  @Test
+  public void testIsOtelMetric() {
+    assertTrue(OtlpTraceUtils.isOtelMetric("process.runtime.jvm.threads.count"));
+    assertTrue(OtlpTraceUtils.isOtelMetric("process.runtime.jvm.memory.usage"));
+    assertTrue(OtlpTraceUtils.isOtelMetric("runtime.jvm.gc.count"));
```

Comment:
For completeness, try testing a value for which isOtelMetric() is false.

Reply:
sure